### ISSUE TITLE
Submit to an evaluation queue documentation

### ISSUE THREAD

**Issue by sieberts** [Sage-Bionetworks/synapseDocs#668]
Because challenge wikis are now fairly stripped down in terms of detailed instructions, we need docs pages that sufficiently explain how to submit to a queue for the purposes of a challenge. 

It would be useful to have this be on it's own page, rather than in the middle of a much longer page that doesn't pertain to most people who view it.

For the purposes of a challenge, it also needs some details about submission as a team (e.g. team must be registered for the challenge). Correct handling of teams is important for tracking submission limits.

We have a hackathon launching on Dec 13th, so would love to see improvements prior to this launch if at all possible.

**Comment by kelshmo** [comment]
@sieberts Does #642 address your needs? You and Tom are the experts when it comes to Challenge docs. If you want to see additional changes implemented, you should meet with Kenny or I.

**Comment by sieberts** [comment]
@kelshmo- No. I wrote this in response to the updates after the PR was merged. @kdaily and I spoke yesterday. He told me to open an issue, so I did.
